tests/test_standardizer.py

```diff
@@ -0,0 +1,357 @@
+            std = Standardizer.get(self.fits, required_flag=False)
+        stdmeta = std.standardizeMetadata()
+        self.assertFalse(stdmeta["required_flag"])
+
+        std = Standardizer.get(self.fits, required_flag=True, optional_flag=True)
+        stdmeta = std.standardizeMetadata()
+        self.assertTrue(stdmeta["required_flag"])
+        self.assertIn("optional_flag", stdmeta)
+        self.assertTrue(stdmeta["optional_flag"])
+
+    def test_instantiation(self):
+        """Test priority, forcing and automatic selection works."""
+        std = Standardizer.get(self.fits, required_flag=True)
+        self.assertIsInstance(std, KBMODV1)
+
+        MyStd.resolveTarget = MyStd.yesStandardize
+        std = Standardizer.get(self.fits, required_flag=True)
+        self.assertIsInstance(std, MyStd)
+
+        MyStd.priority = 0
+        std = Standardizer.get(self.fits, required_flag=True)
+        self.assertIsInstance(std, KBMODV1)
+
+        # Test forcing ignores everything
+        MyStd.resolveTarget = MyStd.noStandardize
+        MyStd.priority = 0
+        std = Standardizer.get(self.fits, force=MyStd, required_flag=True)
+        self.assertIsInstance(std, MyStd)
+
+        # Test instantiating from a single HDUList
+        std = Standardizer.get(self.fits)
+        self.assertIsInstance(std, Standardizer)
+
+        # Test force direct and named
+        std2 = Standardizer.get(self.fits, force=KBMODV1)
+        self.assertIsInstance(std, KBMODV1)
+        self.assertEqual(std2.location, std.location)
+        self.assertEqual(std2.hdulist, std.hdulist)
+
+        std2 = Standardizer.get(self.fits, force="KBMODV1")
+        self.assertIsInstance(std, KBMODV1)
+        self.assertEqual(std2.location, std.location)
+        self.assertEqual(std2.hdulist, std.hdulist)
+
+        # Test from path
+        hdul = FitsFactory.mock_fits(spoof_data=True)
+        tmpf = tempfile.NamedTemporaryFile(suffix=".fits", delete=False)
+        hdul.writeto(tmpf.file, overwrite=True)
+        hdul.close()
+        tmpf.close()
+
+        std2 = Standardizer.get(tmpf.name)
+        self.assertIsInstance(std, KBMODV1)
+        self.assertEqual(std2.location, tmpf.name)
+        self.assertEqual(len(std2.hdulist), 16)
+
+        # clean up resources
+        os.unlink(tmpf.name)
+
+
+# This is in test_standardizeer because totest Standardizer because it's easier
+# than making multiple new standardizers for sake of technical clarity or style
+# Test KBMODV1 more extensively than other standardizers to cover for the
+# possible code-paths through Standardizer itself. TODO: eventually update this
+class TestKBMODV1(unittest.TestCase):
+    """Test KBMODV1 Standardizer and Standardizer."""
+
+    def setUp(self):
+        self.fits = FitsFactory.mock_fits(spoof_data=True)
+
+    def tearDown(self):
+        # Note that np.int32 in setUp is necessary because astropy will raise
```

Comment:
`setUp` doesn't have `np.int32`. Should it?

Reply:
Mocking used to be here, but was moved to `utils`. Moved the comment to an appropriate location there. Adjusted the comments there to include, or better explain really, what I was trying to say here.